README.md

```diff
@@ -1 +1,248 @@
+
+![](acc.png)
+
+_Figure 1: KGs. The image on the left shows the original knowledge graph, while the image on the right demonstrates the updated knowledge graph. The green highlighted box indicates new knowledge that has been added, while the **red highlighted box** indicates any inconsistencies caused by factual changes, i.e., **incorrect date of birth**._
+
+The ACC process will ensure human-centricity is maintained alongside automated validation.
+
+### Provenance
+To enable trust, the provenance, such as the source of the information and the curators (in the case of manual) of all the information, shall be maintained. The provenance conflict resolution mechanism will also be implemented to ensure the accuracy of the provenance information.
+
+### Querying and Reasoning
+BrainKB shall support the KGs' querying and reasoning. It shall also support other downstream analytics tasks, such as link predictions using machine learning techniques.
+
+### Integration and Interoperability
+To ensure interoperability and ease of integration, BrainKB will focus on using standardized ontologies or schemas. However, not all standardized ontologies or schemas are available. In such cases, other schemas or ontologies must be used. To ensure the interoperability, the alignment will be performed where necessary.
+
+### Minimize Cognitive Burden and Data Fatigue
+As BrainKB will also provide features to perform the analytics operation in addition to querying the information (or knowledge), a special emphasis shall be placed on ensuring that the information presented to the user does not cause a cognitive burden and data fatigue. A cognitive burden occurs when the brain must exert more effort to understand information, typically resulting from an overload of visual content. For example, the figure below (left) places more cognitive burden than on the right.
+
+![](cognitive_burden.png)
+
+## Features
+
+## Other considerations
+
+__Assumption:__ We operate on open-world assumptions (OWA), not closed-world assumptions (CSA). In OWA, we do not make any assumptions about the absence of statements, while in CSA absence of statements would be evaluated as false, i.e., assumed to be false.
+
+## Architecture
+The figure below shows the high-level overview of the components of the BrainKB architecture.
+
+![](ingest.png)
+_Figure 2: Ingest Service_
+
+__Application:__ The application (or the application layer) is the go-to point that provides access to BrainKB, such as via UI.
+
+__Service:__ The service layer implements the core logic and is broken down into multiple services based on the functionalities.  
+__Resource:__ The resource will provide the necessary computational resources that are required to deliver the required service by BrainKB.
+
+![](initial-arch.png)
+_Figure 3: Architecture of BrainKB_
+
+## Target Audience
```

Comment:
perhaps you should use future tense in this section

Reply:
updated description